modules/gapi/perf/common/gapi_video_perf_tests_inl.hpp

```diff
@@ -0,0 +1,105 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+#ifndef OPENCV_GAPI_VIDEO_PERF_TESTS_INL_HPP
+#define OPENCV_GAPI_VIDEO_PERF_TESTS_INL_HPP
+
+#include <iostream>
+
+#include "gapi_video_perf_tests.hpp"
+
+namespace opencv_test
+{
+
+  using namespace perf;
+
+//------------------------------------------------------------------------------
+
+#ifdef HAVE_OPENCV_VIDEO
```

Comment:
Does the body of this file really depend on the OPENCV_VIDEO?

As far as I can see, it is not

Reply:
Yes, it does, but indirectly: the function `runOCVnGAPIOptFlowLK()` from `gapi_video_tests_common.hpp` is not defined when `opencv_video` is disabled. Should I add the definition of it (`runOCVnGAPIOptFlowLK(...) {}`) if `opencv_video` is disabled?

Reply:
Pushed my option

Reply:
Ok